app/views/exam_templates/_form.html.erb

```diff
@@ -33,11 +41,18 @@
 
   <div class="exam-template-section">
     <h3><%= ExamTemplate.human_attribute_name('template_divisions') %></h3>
+    <p class="alignright">
+      <%= link_to 'Add Template Division',
+                  '#',
+                  onClick: 'add_template_division(' + exam_template.id.to_s + '); return false;' %>
```

Comment:
When in Ruby code, always prefer string interpolation (with double-quotes, and use `#{...}`).